netsim/tools/nuts/tests/test_bgp.j2

```diff
@@ -0,0 +1,43 @@
+{% set ns = namespace(found=false) %}
+- test_class: TestNapalmBgpNeighbors
+  test_data:
+{% for nn,nd in nodes.items() %}
```

Comment:
If a topology has an FRR container, the tests crash because the the host is in the tests, but not in the inventory. You should check whether `nd.device` is supported (which would be easy if you move the inventory mappings into tool data)

Reply:
FWIW, you check whether the connection is reasonable in the interface checks, but not here.